packages/analytics-browser/src/config.ts

```diff
@@ -68,6 +68,7 @@ export class BrowserConfig extends Config implements IBrowserConfig {
     sessionId?: number,
     public sessionTimeout: number = 30 * 60 * 1000,
     public storageProvider: Storage<Event[]> = new LocalStorage(),
+    public savedMaxCount: number = 1000,
```

Comment:
i think it makes it harder to set a value unless we have a default number that's reasonable and if they want to modify then that's when they'd need to find out how

Reply:
I feel we use a default for local storage (e.g. 1000)
```
init()
```

and configure via storageProvider vs extending the client config. e.g.
```
// user config
init({storageProvider: new LocalStorage({ maxEventCount: 100000 })
```

Other storages might have different limitations. wdyt?

Reply:
local storage can also be used for user session and marketing attribution, which stores plain objects, instead of arrays. so I'm concerned that maxEventCount is specific to a single use case of persisting unsent events